views/Query/index.jsx

```diff
@@ -14,6 +14,7 @@ Query.propTypes = {
     date: React.PropTypes.object,
   }),
   onQueryChange: React.PropTypes.func.isRequired,
+  dateRangeRestricted: React.PropTypes.bool,
```

Comment:
moved this to the appropriate place, missed this one.

I put it in to show a reader that this property could get added to a query object. I /think/ that's appropriate, but am not a React expert.